.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    name: >-
+      ruby ${{ matrix.ruby }}
+    # Available hosts.
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners
+    runs-on: ubuntu-20.04 # focal
+    strategy:
+      matrix:
+        ruby:
+          - '3.0'
```

Comment:
i hate to ask because this is probably some sort of YAML horror, but why is this in quotes and the others are not?

Reply:
Good question. The reason is because when it does not have the quotes, it is recognized as `3` not as `3.0`, while we want the latest version of Ruby 3.0.z there. Here is my experiment. Maybe I saw this behavior in Travis too. Though I am not sure the detail, maybe this issue related to the casing of the value in YAML. The `X.0` is converted to the number. But the `X.1` is converted to the string.
https://github.com/junaruga/mono_logger/runs/2114346195?check_suite_focus=true#step:3:3

This is the case with quotes.
https://github.com/junaruga/mono_logger/actions/runs/618181358

You see this tip on another repo too.
https://github.com/rack/rack/blob/14a236b4f0899e46bc41c4f80dcff29159a59312/.github/workflows/development.yml#L11